src/docks/timelinedock.cpp

```diff
@@ -664,6 +665,15 @@ void TimelineDock::addVideoTrack()
         new Timeline::AddTrackCommand(m_model, true));
 }
 
+void TimelineDock::alignSelectedClips()
+{
+    auto selection = selectionUuids();
+    saveAndClearSelection();
+    AlignAudioDialog dialog(tr("Align clips to audio reference"), &m_model, selection, this);
```

Comment:
Window titles should have each word capitalized. Also, if an action triggers a dialog, usually they should have the same string. In `Clip.qml` this is "Align To Reference Track"

Reply:
done